.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -160,6 +160,8 @@ jobs:
       run: tar -cJf llvm_sycl.tar.xz -C $GITHUB_WORKSPACE/build/install .
     - name: Pack LIT
       run: tar -cJf lit.tar.xz -C $GITHUB_WORKSPACE/src/llvm/utils/lit .
+    - name: Pack End-to-End LIT tests
+      run: tar -cJf e2e_tests.tar.xz -C $GITHUB_WORKSPACE/src/sycl/test-e2e .
```

Comment:
Do we use lit in other tasks to run tests other than e2e?
It will be better to have a single archive for running LIT tests on a different machine.

Another option to consider is to checkout LIT and e2e from sources. Currently `devops/actions/llvm_test_suite/action.yml` does `git checkout` for `llvm-test-suite` repository, but after we move the tests we have two options:
- pass them as artifacts
- checkout PR sources

IMHO it's easier to checkout sources. This way requires less GitHub resources, which are quite limited.

The only caveat is that I don't see if actions/checkout allows to checkout one-two directories instead of whole repository.